cluster/common.sh

```diff
@@ -603,6 +603,7 @@ KUBELET_TOKEN: $(yaml-quote ${KUBELET_TOKEN:-})
 KUBE_PROXY_TOKEN: $(yaml-quote ${KUBE_PROXY_TOKEN:-})
 ADMISSION_CONTROL: $(yaml-quote ${ADMISSION_CONTROL:-})
 MASTER_IP_RANGE: $(yaml-quote ${MASTER_IP_RANGE})
+MASTER_RESERVED_IP: $(yaml-quote ${MASTER_RESERVED_IP})
```

Comment:
Jerzy, the question is not how it will affect HA master mode, which can be more expensive and this cost estimate have been already covered in the PRD. The question is what will be the increase in cost in single master mode, if we start using external IP address. Can you please update the spreadsheet with this computation? I think currently it's answering the question that we need to answer.